draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -318,7 +318,7 @@
    after message parsing has delineated the individual field lines.
 </t>
 <t>
-   Although the line terminator for the start-line and header
+   Although the line terminator for the start-line and
```

Comment:
```suggestion
   Although the line terminator for the start-line and header
```

Reply:
Ah - no, this isn't specific to header fields, it's also true for trailer fields. I.e., this text is not about the delimiter between the start-line and header fields, it's about the *line terminator* that the start-line and fields share.

Reply:
Right.